docs/ARCHITECTURE.md

```diff
@@ -0,0 +1,100 @@
+[comment]: <> "LTeX: language=en-US"
+
+# Architecture Decisions, Automations and Contributions
+
+This file describes all used frameworks and packages, the reasoning behind them and all other architectural decisions as also used automations and all contributions made during this project.
+
+---
+
+## Packages and Frameworks
+
+### Deno
+
+We use [deno.land](https://deno.land), the evolution of Node.js, as runtime for our TypeScript backend. Compared to [Node.js](https://nodejs.org/en/), Deno allows for much greater granularity when it comes to security permissions and has a good *secure-by-default*-ruleset to limit what applications can do if no special permissions given.
+
+### Svelte
+
+[Svelte](https://svelte.dev/) is a lightweight, modern and fast frontend framework. Svelte has some advantages for which we chose it over other frameworks [source](https://daily.dev/blog/building-with-svelte-all-you-need-to-know-before-you-start):
+
+- Instead of doing most of the work in the browser, Svelte does its work when you build the app, and it compiles it to efficient vanilla JavaScript.
+- It reduces the overhead of the JavaScript framework you’re using. Compiling Svelte to vanilla JavaScript improves the code readability, enables re-use, and produces a much faster web app.
+- Svelte does not require declarative, state-driven code, which the browser has to convert into DOM operations. That means you don’t need to use virtual DOM anymore.
+
+### Snel
+
+[Snel](https://deno.land/x/snel@v0.6.0) is a tool/framework to compile Svelte components to JavaScript files using Deno and Svelte to create web applications.
```

Comment:
```suggestion
[Snel](https://deno.land/x/snel) is a tool/framework to compile Svelte components to JavaScript files using Deno and Svelte to create web applications.
```
I would prefer to link to the latest version of all tools and frameworks in the documentation